pkgs/by-name/hi/hidviz/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, stdenv
+# Package dependencies
+, qt6
+, libusb1
+, protobuf
+, asio
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hidviz";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "hidviz";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-9crHFYVNNxJjwJojwqB8qdAGyr1Ieux9qC3m3rpIJw0=";
+  };
+
+  dontWrapQtApps = true;
+
+  preConfigure = ''
+    substituteInPlace libhidx/cmake_modules/Findasio.cmake --replace-fail '/usr/include/asio' '${lib.getDev asio}/include/asio'
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    qt6.qttools
+  ];
+  buildInputs = [
+    qt6.qtwebengine
+    libusb1
+    protobuf
+    asio
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/hidviz/hidviz";
+    description = "A GUI application for in-depth analysis of USB HID class devices.";
+    license = licenses.gpl3;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [];
```

Comment:
Would you be willing to add yourself as a maintainer as well?

You can add it in this PR in a separate commit if you would like!

We prefer not to merge packages without a listed maintainer, especially leaf packages, since there will be no one to ping for questions or for testing PRs

Reply:
It's unfortunate, but not wanting to put yourself down is understood <3

Is there anyfew you know of who might use the package and be interested to at least test package bumps and such? (e.g. if r-ryantm makes a bump PR, listed maintainers can test and have the mergebot auto-merge this leaf package)